src/parsers/parser_nodes.ts

```diff
@@ -26,25 +27,30 @@ export class ParsedRange {
 
 // export type ParsedNodeType = 'expect' | 'describe' | 'it' | 'ROOT';
 
-export const ParsedNodeTypes = {
+export const ParsedNodeTypes: {
+  describe: 'describe';
+  expect: 'expect';
+  it: 'it';
+  root: 'root';
+} = {
   describe: 'describe',
   expect: 'expect',
   it: 'it',
   root: 'root',
 };
 
-export type ParsedNodeType = $Keys<typeof ParsedNodeTypes>;
+export type ParsedNodeType = keyof typeof ParsedNodeTypes;
 
 export class ParsedNode {
   type: ParsedNodeType;
 
-  start: Location;
+  start: Location | undefined = undefined;
 
-  end: Location;
+  end: Location | undefined = undefined;
 
   file: string;
 
-  children: ?Array<ParsedNode>;
+  children?: Array<ParsedNode>;
```

Comment:
`ParsedNode[]` is easier to read than `Array<...>`.

Reply:
I agree.  Pretty sure this is a Prettier/ESLint rule.